src/Serilog.Sinks.MongoDB/project.json

```diff
@@ -0,0 +1,21 @@
+﻿{
+  "version": "2.0.0-beta-*",
+  "description": "The MongoDB sink for Serilog",
+  "authors": [ "Kiran Makkapati", "Serilog Contributors" ],
+  "tags": [ "serilog", "mongodb" ],
+  "projectUrl": "http://serilog.net",
+  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+  "iconUrl": "http://serilog.net/images/serilog-sink-nuget.png",
+  "dependencies": {
+    "Serilog": "2.0.0-beta-511",
+    "Serilog.Sinks.PeriodicBatching": "2.0.0-beta-700"
+  },
+
+  "frameworks": {
+    "dnx46": {
```

Comment:
:+1:

Since this is now on `dev` we'll need a separate commit/PR to cover it. @brantw it'd be great if you can take a look to ensure whatever new TFM is chosen works in the projects you're using this on. Any and all help appreciated :-)